### ISSUE TITLE
Migrate price quotes from CoinGecko to Moralis (6 hours)

### ISSUE THREAD

**Issue by orbitmechanic** [orbitmechanic/USAMobile#147]
The reliability in matching CoinGecko symbols to contract addresses is very low.  It is too easy to get the token symbol at CoinGecko wrong.  Consider  the (POS) Decentraland behavior in these screen shots.  It's not really priced like that.  And it comes out as "Matic" from the 1Inch swap.
![Screen Shot 2021-11-15 at 7 35 49 AM](https://user-images.githubusercontent.com/56613757/141783385-07eab466-d551-4a5f-8ec7-3287fb45e270.png)
![Screen Shot 2021-11-15 at 7 32 46 AM](https://user-images.githubusercontent.com/56613757/141783398-fe15bfcf-baac-4d64-8ee8-d4048935d8f7.png)
Switch our price oracle to just Moralis [getTokenPrice](https://docs.moralis.io/moralis-server/web3-sdk/token#gettokenprice).  We may be stuck with CoinGecko for native prices because they don't have a "getNativePrice" function yet.

**Comment by git-tn** [comment]
Migrated from coingecko to moralis API, but it seems their API has few issues.

First, we can't get native token price. their getTokenPrice uses token address to get prices, so I tried to use  nativePrice field from API response but it's actually returning ETH values, not MATIC or issued token price. so I'm using coingecko for native token prices.
Second, moralis APIs are not returning good metadata, their API don't return thumb, logos right, so I cached the 1inch API response for this usage.
